### PR TITLE
refactor such that tables and views share schema logic

### PR DESCRIPTION
**Pull request by john-hill**

### CODE REVIEW

File: services/repository/src/main/java/org/sagebionetworks/repo/web/service/table/TableServicesImpl.java [hunk 1]
```diff
@@ -272,4 +268,13 @@ public SnapshotResponse createTableSnapshot(Long userId, String tableId, Snapsho
 		return tableEntityManager.createTableSnapshot(userInfo, tableId, request);
 	}
 	
+	@Override
+	public TableBundle getTableBundle(IdAndVersion idAndVersion) {
+		ValidateArgument.required(idAndVersion, "idAndVersion");
+		List<ColumnModel> tableSchema = columnModelManager.getColumnModelsForObject(idAndVersion);
```
Comment (john-hill) [inline services/repository/src/main/java/org/sagebionetworks/repo/web/service/table/TableServicesImpl.java:274]: missing auth read check on the get columns for table

File: services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/ColumnModelManagerImpl.java [hunk 2]
```diff
@@ -426,14 +431,48 @@ public boolean truncateAllColumnData(UserInfo user) {
 		return details;
 	}
 
+	/**
+	 * The schema for the current version for any table/view only exists in the
+	 * database with a null version number. Therefore, calls to get the schema of
+	 * the current version are transformed to a representation with a a null
```
Comment (john-hill) [inline services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/table/ColumnModelManagerImpl.java]: a a

File: services/workers/src/test/java/org/sagebionetworks/table/worker/TableCSVAppenderWorkerIntegrationTest.java [hunk 3]
```diff
@@ -70,7 +70,7 @@
 @ContextConfiguration(locations = { "classpath:test-context.xml" })
 public class TableCSVAppenderWorkerIntegrationTest {
 
-	public static final int MAX_WAIT_MS = 1000 * 60;
+	public static final int MAX_WAIT_MS = 1000 * 60*1000;
```
Comment (john-hill) [inline services/workers/src/test/java/org/sagebionetworks/table/worker/TableCSVAppenderWorkerIntegrationTest.java]: undo this change

**Review by marcomarasca — APPROVED** [review]
LGTM
